x/gov/simulation/operations.go

```diff
@@ -420,7 +420,7 @@ func randomProposalID(r *rand.Rand, k keeper.Keeper,
 	}
 
 	proposal, ok := k.GetProposal(ctx, proposalID)
-	if !ok || proposal.Status != status {
+	if !ok || v1beta1.ProposalStatus(proposal.Status) != status {
```

Comment:
is there some plan to update simulation operations to use v1beta2?

Reply:
Yes!